test_scripts/Defects/8_1/980_SDL_responds_GENERIC_ERROR_instead_of_INVALID_DATA_when_soft_button_has_Type_is_Image_or_Both_and_Text_is_whitespace_or_\t_or_\n_or_empty.lua

```diff
@@ -0,0 +1,81 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Please use not so long branch name. E.g. `issue/980`
Please also make script file name shorter.
Please try to avoid `\` or `/` in file names. There are a few more not recommended characters (e.g. ` `, `,` etc.) 

It's really hard to handle such long strings